pubspec.yaml

```diff
@@ -47,6 +47,7 @@ dependencies:
   shimmer: ^2.0.0
   email_validator: '^2.1.16'
   flutter_secure_storage: ^8.0.0
+  collection: ^1.16.0
```

Comment:
How about using the latest version `1.17.1`

Reply:
@chornerman I tried the latest version first but it produces pub error,also tried 1.17.0, it also produced error

Reply:
@Shayokh144 Oh I see